ui/raidboss/data/05-shb/trial/wol.txt

```diff
@@ -6,11 +6,13 @@
 hideall "--sync--"
 hideall "--Reset--"
 
-0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
-
 ### Phase 1: 100%->50%
 # Note: Don't sync on engage or auto here because there's a checkpoint.
+# Can use InCombat line to start timeline since the multiple phases do not
+# change the player's InCombat status.
```

Comment:
^ this note, in that there's a checkpoint here

(also randomly, it looks like the sync at t=490 should have a larger window??)